modules/plugins/languages/typst.nix

```
@@ -167,6 +167,48 @@ in {
           };
         };
       };
+      typst-concealer = {
+        enable =
+          mkEnableOption ''
+            [typst-concealer]: https://github.com/PartyWumpus/typst-concealer
+
+            Inline typst preview for Neovim via [typst-concealer]
+          ''
+          // {default = false;};
```

```suggestion
          mkEnableOption ''
            [typst-concealer]: https://github.com/PartyWumpus/typst-concealer

            Inline typst preview for Neovim via [typst-concealer]
          '';
```

mkEnableOption defaults to false.